Program.fs

```diff
@@ -79,20 +82,32 @@ let findInstalledPackages (packageJson: string) : ResizeArray<InstalledNpmPackag
     let dependencies : JProperty list = [
         if content.ContainsKey "dependencies"
         then yield! (content.["dependencies"] :?> JObject).Properties() |> List.ofSeq
```

Comment:
When adding `Thoth.Json.Net` to `Femto` I saw that you used `Newtonsoft` but didn't find the place (at the time).

But as I now see where it's being used I take this opportunity to ask this question:

I think we should make it consistent how we access the JSON. Strangely ^^ I am in favour of using `Thoth.Json.Net` everywhere but if you don't agree we can use `Newtonsoft` :)

Reply:
I find `Newtonsoft` easier to work with because I am more familiar with it and have been using it for a very long time. I don't think it makes a lot of difference to use `Thoth.Json.Net` but I guess it is more "functional" and more "elegant" than the low-level `Newtonsoft`, so might as well just use `Thoth.Json.Net` everywhere ;) 

See #36